### PR TITLE
Connection Pool: Connection Pool Size

### PR DESCRIPTION
**Pull request by Vavilen**
Fix [#653](https://github.com/skycoin/skycoin/issues/653)

Just a notice:
In fact, connections count can be exceeded at first seconds after the application start and be remain at ~ constant level, because outgoing connections count is checked only here:
```
// Fill up our outgoing connections
		case <-outgoingConnectionsTicker:
```
but there is other case with peer connecting:
`case <-privateConnectionsTicker:`

### CODE REVIEW

**Comment by iketheadore** [comment]
Change outgoing connection pool size won't fix the peer list full issue. We need to change the pool size of incoming connection pool size, and it's default value 1000, managed in src/daemon/pex/pex.go.

**Comment by Vavilen** [comment]
@iketheadore, ok, I`ll add this param as command line param. What is the best default value for this parameter?

File: cmd/skycoin/skycoin.go [hunk 1]
```diff
@@ -486,6 +490,7 @@ func configureDaemon(c *Config) daemon.Config {
 	dc := daemon.NewConfig()
 	dc.Pex.DataDirectory = c.DataDirectory
 	dc.Pex.Disabled = c.DisablePEX
+	dc.Pex.Max = c.MaxIncomingConnections
```
Comment (gz-c) [inline cmd/skycoin/skycoin.go]: Is this the right setting to apply? Or do we need a new setting just for peer list size?

Comment (gz-c) [reply cmd/skycoin/skycoin.go]: The peerlist size controls the max incoming connections?

Comment (iketheadore) [reply cmd/skycoin/skycoin.go]: I think need a new setting, the peer list doesn't control the incoming connections directly.

**Comment by gz-c** [comment]
#709

**Pull request closed without merging**